src/games/strategy/performance/PerfTimer.java

```diff
@@ -64,11 +68,16 @@ public static PerfTimer startTimer(String title) {
     }
   }
 
-  protected static void processResult(long stopNanos, PerfTimer perfTimer) {
+  private static void processResult(long stopNanos, PerfTimer perfTimer) {
     long stopMicros = stopNanos / 1000;
 
     long milliFraction = (stopMicros % 1000) / 100;
     long millis = (stopMicros / 1000);
     PerformanceConsole.getInstance().append(millis + "." + milliFraction + " ms - " + perfTimer.title + "\n");
   }
-}
\ No newline at end of file
+
+  @Override
```

Comment:
Could you group this method closer to the 'stop' method, this way related entities/methods are closer together.